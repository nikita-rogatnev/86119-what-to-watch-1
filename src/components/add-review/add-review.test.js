import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import AddReview from './add-review';

it(`AddReview renders correctly`, () => {
  const tree = renderer.create(<BrowserRouter><AddReview/></BrowserRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
