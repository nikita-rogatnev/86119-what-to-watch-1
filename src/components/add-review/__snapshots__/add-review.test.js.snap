// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddReview renders correctly 1`] = `
<div
  className="add-review"
>
  <form
    action="#"
    className="add-review__form"
  >
    <div
      className="rating"
    >
      <div
        className="rating__stars"
      >
        <input
          className="rating__input"
          id="star-1"
          name="rating"
          type="radio"
          value="1"
        />
        <label
          className="rating__label"
          htmlFor="star-1"
        >
          Rating 1
        </label>
        <input
          className="rating__input"
          id="star-2"
          name="rating"
          type="radio"
          value="2"
        />
        <label
          className="rating__label"
          htmlFor="star-2"
        >
          Rating 2
        </label>
        <input
          checked=""
          className="rating__input"
          id="star-3"
          name="rating"
          type="radio"
          value="3"
        />
        <label
          className="rating__label"
          htmlFor="star-3"
        >
          Rating 3
        </label>
        <input
          className="rating__input"
          id="star-4"
          name="rating"
          type="radio"
          value="4"
        />
        <label
          className="rating__label"
          htmlFor="star-4"
        >
          Rating 4
        </label>
        <input
          className="rating__input"
          id="star-5"
          name="rating"
          type="radio"
          value="5"
        />
        <label
          className="rating__label"
          htmlFor="star-5"
        >
          Rating 5
        </label>
      </div>
    </div>
    <div
      className="add-review__text"
    >
      <textarea
        className="add-review__textarea"
        id="review-text"
        name="review-text"
        placeholder="Review text"
      />
      <div
        className="add-review__submit"
      >
        <button
          className="add-review__btn"
          type="submit"
        >
          Post
        </button>
      </div>
    </div>
  </form>
</div>
`;
